src/serializer.js

```diff
@@ -915,7 +955,9 @@ var Serializer = (function () {
     }
     for (var ns in namespaceCounts) {
       if (!namespaceCounts.hasOwnProperty(ns)) continue
-      str += '\n xmlns:' + this.prefixes[ns] + '="' + escapeForXML(ns) + '"'
+      // Rel uris in xml ns is not strictly allowed in the spec but needed in practice often
+      var ns2 = (this.base && this.flags.indexOf('Z') < 0) ? Util.uri.refTo(this.base, ns) : ns
```

Comment:
The people who agreed on the XML namespaces spec insisted on two things against my better judgement.  One was "IT is not a goal to ereference namespace URIs" (which was just unhelpful" and that "all namespace URIs must be absolute".   The problem was the NS spec was done RDF people by XML people who didn't get RDF needs. In the cwm code, a special "z" flag ion the serializer allowed one to break this rule, which is sometimes really valuable.   Here I switched to a "Z" (cap) flag if you want to be strict, default is break the rule. - flip case, flip sense.  We could offer a requested change to the spec at this late date o we could publish a set of spec changes assumed in solid.